app/lib/messages.js

```diff
@@ -26,10 +27,35 @@ function searchInfomationMessage(singleResult, searchPostcode, searchTerm) {
   }
 
   if (searchTerm) {
-    returnValue += `${(searchPostcode) ? ' and' : ''} using '${searchTerm}'`;
+    returnValue += ` using '${searchTerm}'`;
   }
+  return returnValue;
+}
 
-  return `${returnValue}.`;
+function searchInfomationMessage(singleResult, searchPostcode, searchTerm) {
+  return `We found ${(singleResult) ? 'this surgery' : 'these surgeries'}${promptBuilder(searchPostcode, searchTerm)}.`;
+}
+
+function noResultsMessage(res, noResult, searchPostcode, searchTerm) {
+  let headerPrompt = '';
+  let paragraphPrompt = '';
+  if (noResult) {
+    headerPrompt = `<h2>We can not find a surgery${promptBuilder(searchPostcode, searchTerm)}</h2>`;
```

Comment:
I'm uncomfortable about this side effect. We shouldn't be returning the noResultsMessage and setting a res.locals value. An alternative would be to return an object (e.g. { header: string, paragraph: string, class: string })